### PR TITLE
Support int type for eventPid and eventTid fields

### PR DESCRIPTION
**Pull request by nagworld9**
## Description
Today we only support string value for eventPid and EventTid. As these fields are ids and ids are generally numeric in nature, so changing logic to support int values too. 

Issue # 

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azurelinuxagent/ga/collect_telemetry_events.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ class ExtensionEventSchema(object):
            "EventTid":"2",
            "OperationId":"Guid (str)"
         }
+
+    From next version(2.10+) we accept integer values for EventPid and EventTid fields. But we still support string type for backward compatability
```
Comment (nagworld9) [inline azurelinuxagent/ga/collect_telemetry_events.py:62]: Assumed this go in next release and version would be 2.10

File: azurelinuxagent/ga/collect_telemetry_events.py [hunk 2]
```diff
@@ -323,15 +325,20 @@ def _parse_event_and_ensure_it_is_valid(self, extension_event):
         :param extension_event: The json event from file
         :return: Verified Json event that qualifies the contract.
         """
-
-        clean_string = lambda x: x.strip() if x is not None else x
+        def _clean_value(k, v):
+            if v is not None:
```
Comment (narrieta) [inline azurelinuxagent/ga/collect_telemetry_events.py:329]: if v is None, _clean_value will return None and the conversion below will raise an exception

Comment (nagworld9) [reply azurelinuxagent/ga/collect_telemetry_events.py:329]: Sorry, where? I kept it same as we have today so that downstream code will work like before.

Comment (narrieta) [reply azurelinuxagent/ga/collect_telemetry_events.py:329]: never mind, you are converting the key, not the value

**Review by narrieta — APPROVED** [review]

**Review by maddieford — APPROVED** [review]

**Pull request merged**